plugin-manager/src/main/kotlin/com/itsaky/androidide/plugins/manager/services/IdeBuildServiceImpl.kt

```
@@ -45,6 +48,19 @@ class IdeBuildServiceImpl private constructor() : IdeBuildService {
         buildStatusListeners.remove(callback)
     }
 
+    override fun executeTasks(vararg tasks: String): CompletableFuture<Boolean> {
+        val buildService = Lookup.getDefault().lookup(BuildService.KEY_BUILD_SERVICE)
+            ?: return CompletableFuture.completedFuture(false)
+
+        return try {
+            buildService.executeTasks(*tasks)
+                .thenApply { result -> result?.isSuccessful == true }
+                .exceptionally { false }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)